packages/es-components/src/components/base/icons/Icon.js

```diff
@@ -6,7 +6,7 @@ import { regular } from './icon-definitions';
 
 const StyledIcon = styled.i`
   display: inline-block;
-  font-family: ${props => props.fontFamily} !important;
```

Comment:
Why did we want to have the font family specified by the props again? I can't remember why we supported this as a prop.

Reply:
I'm not sure. The value was hard-coded anyway and then passed to this styled component, so putting the hard-coded value here just saves us a line of code.

Reply:
Sure, `fontFamily`'s initial value was always hard-coded into the object, but it was always possible to overwrite it with props. This was just never documented.

Reply:
Ah yes, I didn't catch that it could be overridden by the `...other` props. Do you think we should still allow that bypass?

Reply:
Personally I rather do away with the bypass. We can easily argue that it was never a documented feature and going forward we want to standardize the usages of the `indv-mkt-icons` font family.